content/talks/responsive-components.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Container Queries & The Future of CSS
+sub: A path towards responsive components & design systems
+author: miriam
+image:
+  src: talks/vars-cascade.jpg
+  width: 2898
+  height: 1522
+tags:
+  - Cascade
+  - CSS
+  - Container Queries
+  - CSS Scope
+  - Cascade Layers
+  - CSSWG
+events:
+  - venue: Web Directions, Hover
+    url: https://webdirections.org/hover/
+    date: 2021-04-09
+    end: 2021-04-30
+    adr: Online
+  - venue: Modern Web Conference
+    url: https://modernwebconference.com/
+    date: 2021-05-17
+    end: 2021-05-28
+    adr: Online
+summary: |
+  From Container Queries
+  to Cascade Layers and (likely) Scoped Styles,
+  a number of new CSS specifications
+  are aimed at modernizing the cascade
+  and making fully responsive components 
+  and design systems a reality.
+---
+
+Over the last decade
+Object-Oriented & Responsive design
+have become the norm --
+with tools like flexbox, grid, and aspect-ratio
```

Comment:
maybe "Flexbox, Grid, and `aspect-ratio`"? 

Everything else is clear and reads great to me.

Reply:
```suggestion
with tools like Flexbox, Grid, and `aspect-ratio`
```